File: Level-0/37.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        for(int i = 0; i < rsp.length(); i++){
            char c = rsp.charAt(i);
            if(c == '2') answer += '0';
            if(c == '0') answer += '5';
            if(c == '5') answer += '2';
        }
        return answer;
    }
}

File: Level-2/37.js
function solution(progresses, speeds) {
    const answer = [];
    const arr = [];
    for(let i = 0; i < progresses.length; i++) {
        arr[i] = Math.ceil((100 - progresses[i]) / speeds[i]);
    }
    
    let cnt = 1;
    let day = arr[0];
    for(let i = 1; i < arr.length; i++) {
        if(arr[i] <= day) {
            cnt++;
        }else {
            answer.push(cnt);
            day = arr[i];
            cnt = 1;
        }
    }
    answer.push(cnt);
    
    return answer;
}
